stable/datadog/README.md

```diff
@@ -222,19 +175,101 @@ podLabelsAsTags:
   release: helm_release
 ```
 
+then upgrade your Datadog Helm chart:
+
+```bash
+helm upgrade -f datadog-values.yaml <RELEASE_NAME> stable/datadog --recreate-pods
+```
+
 ### CRI integration
 
-As of the version 6.6.0, the Datadog Agent supports collecting metrics from any container runtime interface used in your cluster.
-Configure the location path of the socket with `datadog.criSocketPath` and make sure you allow the socket to be mounted into the pod running the agent by setting `datadog.useCriSocketVolume` to `True`.
+As of the version 6.6.0, the Datadog Agent supports collecting metrics from any container runtime interface used in your cluster. Configure the location path of the socket with `datadog.criSocketPath` and make sure you allow the socket to be mounted into the pod running the agent by setting `datadog.useCriSocketVolume` to `True`.
 Standard paths are:
 
 - Containerd socket: `/var/run/containerd/containerd.sock`
 - Cri-o socket: `/var/run/crio/crio.sock`
 
-## Updating
+## All configuration options
 
-### From < 1.19.0 to >= 1.19.0
+The following table lists the configurable parameters of the Datadog chart and their default values. Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
```

Comment:
For https://docs.datadoghq.com/agent/kubernetes/helm we discussed we would not recommend using `--set` except for the one-liner install, should we just talk about the values.yaml file instead?

Reply:
idk, this is why i put it at the end of the readme. Since not all options are in the value.yaml file I thought it was nice to have everything here..

But we could remove this table and put everything in the `value.yaml` file, generating a summary table in the DD doc only. What do you think?

Reply:
- Unfortunately, the table [is required](https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#documentation)
- Ideally, all options should be in `values.yaml` for consistency. We'll then work on auto-generating the table via a script
- I was talking about the use of manual `--set`. The table is still relevant when using `values.yaml`, as a bird's eye overview